gbn/gbn_conn.go

```diff
@@ -507,6 +713,8 @@ func (g *GoBackNConn) receivePacketsForever() error { // nolint:gocyclo
 			g.pongTicker.Pause()
 		}
 
+		g.resendTicker.Reset(g.resendTimeout)
```

Comment:
Do you agree that it makes sense to always reset the resend ticker if we receive any type of message? My reasoning behind this, is that if the other party actually sends something to us, we can be sure that they're actually still active. It might just be that the thread that's currently being prioritised for the other party currently, is the one that's resending the queue.

Reply:
yeah makes sense 👍